src/index.ts

```diff
@@ -125,6 +125,8 @@ async function updateToolbar(
           debug.session.debuggedClients.add(debug.session.client.path);
           await debug.start();
         } else {
+          await debug.restart();
+          await debug.clearBreakpoints();
```

Comment:
Not sure we should clean the breakpoints separately, as this looks like a special case.